src/Components/SingleReview.jsx

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useState } from "react";
+import { useParams, Link } from "react-router-dom";
+import { getReviewById } from "../utils";
+
+export const SingleReview = () => {
+  const { review_id } = useParams();
+
+  const [singleReview, setSingleReview] = useState({});
+  const [reviewExists, setReviewExists] = useState(true);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+    getReviewById(review_id)
+      .then((data) => {
+        setReviewExists(true);
+        setSingleReview(data.review);
+        setIsLoading(false);
+      })
+      .catch(() => {
```

Comment:
Good that your thinking ahead, but if it is not being used yet please dont include it.